docs/source/onedal/algorithms/nearest-neighbors/knn-classification.rst

```diff
@@ -69,7 +69,9 @@ m`, by performing the following steps:
 
 #. Identify the set :math:`N(x_j') \subseteq X` of the :math:`k` feature vectors
    in the training set that are nearest to :math:`x_j'` with respect to the
-   Euclidean distance.
+   Euclidean distance chosen by default. Used distance can be customised with
+   predefined set of pairwise distances: Minkowski distances with fractional
+   degree(including Euclidean distance) and Chebyshev distance.
```

Comment:
```suggestion
   Euclidean distance, which is chosen by default. The distance can be customized with
   the predefined set of pairwise distances: Minkowski distances with fractional
   degree (including Euclidean distance) and Chebyshev distance.
```

degree  or degree**s**?

Reply:
Need reference for Minkowski distances and Chebyshev distance

Reply:
applied, please check